integration/setup/setup.go

```diff
@@ -92,8 +92,6 @@ func installAWSOperator(ctx context.Context, config Config) error {
 						Service: chartvalues.AWSOperatorConfigSecretAWSOperatorSecretYamlService{
 							AWS: chartvalues.AWSOperatorConfigSecretAWSOperatorSecretYamlServiceAWS{
 								AccessKey: chartvalues.AWSOperatorConfigSecretAWSOperatorSecretYamlServiceAWSAccessKey{
-									ID:     env.GuestAWSAccessKeyID(),
-									Secret: env.GuestAWSAccessKeySecret(),
```

Comment:
When this is not needed or used anymore we need to check if we can cleanup the template.

Reply:
in here you mean? 

`giantswarm/aws-operator/vendor/github.com/giantswarm/e2etemplates/pkg/chartvalues/aws_operator_template.go`

Reply:
Don't know. A link I can click on would be useful. Need to check.

Reply:
See https://github.com/giantswarm/e2etemplates/blob/33b1a1239b499d811a7bf4391264eb663943784d/pkg/chartvalues/aws_operator_template.go#L49-L56. I am not sure how it should actually be.